ape_safe/client.py

```diff
@@ -75,17 +90,36 @@ class UnexecutedTxData(BaseModel):
     modified: datetime
     safeTxHash: SafeTxID
     confirmationsRequired: int
-    confirmations: List[SafeTxConfirmation]
-    trusted: bool
+    confirmations: List[SafeTxConfirmation] = []
+    trusted: bool = True
     signatures: Optional[HexBytes] = None
 
     @classmethod
-    def from_safe_tx(cls, safe_tx: SafeTx) -> "UnexecutedTxData":
-        return cls(  # type: ignore[arg-type]
+    def from_safe_tx(cls, safe_tx: SafeTx, confirmations_required: int) -> "UnexecutedTxData":
+        return cls(
             safe=safe_tx._verifyingContract_,
-            **safe_tx,
+            submissionDate=datetime.now(),
```

Comment:
**question**: Should this be using the UTC timezone? I think this will localize otherwise.

```python
from datetime import timezone

datetime.now(timezone.utc)
```

Reply:
done